spec/active_record/monetizable_spec.rb

```diff
@@ -121,18 +132,16 @@ class SubProduct < Product
       end
 
       context "when MoneyRails.raise_error_on_money_parsing is true" do
-        before { MoneyRails.raise_error_on_money_parsing = true }
+        before do
+          MoneyRails.raise_error_on_money_parsing = true
+          Money.default_bank.add_rate("RUB", "USD", 1)
+        end
+
         after { MoneyRails.raise_error_on_money_parsing = false }
 
         it "raises exception when a String value with hyphen is assigned" do
           expect { product.accessor_price = "10-235" }.to raise_error ArgumentError
         end
-
-        it "raises an exception if it can't change currency" do
-          expect {
-            Product.new.price = Money.new(10, "RUB")
-          }.to raise_error(MoneyRails::ActiveRecord::Monetizable::ReadOnlyCurrencyException, "Can't change readonly currency 'USD' to 'RUB' for field 'price'")
-        end
```

Comment:
I have rethought this part. The code is a bit hard to read, so please correct me if I've misunderstood the lines:
The ReadOnlyCurrencyException will be raised, when a field has another currency than the assigned one? 
But because of the conversion beforehand
```
if money_currency && current_currency != money_currency.id
```
can't be true anymore, because current_currency and money_currency are always equal.
So this part can be removed?